test/test-suite.js

```diff
@@ -593,15 +593,15 @@ module.exports = function (DOMPurify, window, tests, xssTests) {
     );
     assert.equal(
       DOMPurify.sanitize(
-        '<foo-bar baz="foobar"></foo-bar><div is="foo-baz"></div',
-        { ALLOWED_CUSTOM_ELEMENTS: (tagName) => tagName.startsWith('foo-') }
+        '<foo-bar bas="foobar"></foo-bar><div is="foo-baz"></div>',
+        { ALLOWED_CUSTOM_ELEMENTS: (tagName) => tagName.match(/^foo-/) }
       ),
       '<foo-bar baz="foobar"></foo-bar><div is="foo-baz"></div>'
     );
     assert.equal(
       DOMPurify.sanitize(
-        '<foo-bar baz="foobar"></foo-bar><div is="foo-baz"></div>',
-        { ALLOWED_CUSTOM_ELEMENTS: (tagName) => tagName.endsWith('-bar') }
+        '<foo-bar bas="foobar"></foo-bar><div is="foo-baz"></div>',
```

Comment:
`bas` should be `baz`

Reply:
Should I fix real quick?

Reply:
Yes, please. I'd be happy not to have to do another fork and PR for this.